tasks/request_files/test/unit_tests/test_request_files.py

```diff
@@ -1217,18 +1244,18 @@ def test_task_one_granule_1_file_db_error(
     @patch("boto3.client")
     @patch("cumulus_logger.CumulusLogger.info")
     def test_task_file_not_in_glacier(
-            self,
-            mock_logger_info: MagicMock,
-            mock_boto3_client: MagicMock,
-            mock_post_entry_to_queue: MagicMock,
+        self,
+        mock_logger_info: MagicMock,
+        mock_boto3_client: MagicMock,
+        mock_post_entry_to_queue: MagicMock,
     ):
         """
         Test a file that is not in glacier.
         # todo: Expand test descriptions.
         """
         dest_bucket = uuid.uuid4().__str__()
         file1 = "MOD09GQ___006/2017/MOD/MOD09GQ.A0219114.N5aUCG.006.0656338553321.xyz"
-        granule_id = "MOD09GQ.A0219114.N5aUCG.006.0656338553321"
+        granule_id = "MOD09GQ.A0219114.N5aUCG.006.0656338553321.xyz"
```

Comment:
I thought granules didn't have extensions, just the files. Am I wrong?

Reply:
you are right. 
Fixed